dashboard/test/helpers/levels_helper_test.rb

```diff
@@ -247,6 +247,42 @@ def request
     assert_not_equal channel, get_channel_for(level, script.id)
   end
 
+  test 'uses_google_blockly is false if not set' do
+    @level = build :level
+    refute use_google_blockly
+  end
+
+  test 'use_google_blockly is true if useGoogleBlockly is set in view_options' do
+    view_options(useGoogleBlockly: true)
+    @level = build :level
+    assert use_google_blockly
+
+    reset_view_options
+  end
+
+  test 'use_google_blockly is true if level.uses_google_blockly?' do
+    Level.any_instance.stubs(:uses_google_blockly?).returns(true)
+    @level = build :level
+    assert use_google_blockly
+
+    Level.unstub(:uses_google_blockly?)
+  end
+
+  test 'use_google_blockly is false if level.uses_google_blockly? but disable_google_blockly is set' do
+    GamelabJr.any_instance.stubs(:uses_google_blockly?).returns(true)
+    @level = build :spritelab
+    assert use_google_blockly
+
+    DCDO.stubs(:get).with('disable_google_blockly', []).returns(['GamelabJr'])
```

Comment:
I think the DCDO array values can be all lowercase?  (Similar comment for example in the PR description.)

Reply:
they can -- i added this case and the one after it to display that these DCDO values are case insensitive, but i think it's clearer to display that the strings in this array should match the Level subclass name, so i'd like to leave the casing consistent even though either way will work. i'll add a note to the PR description about it though